subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -815,6 +815,13 @@ config BT_CTLR_USER_CPR_INTERVAL_MIN
 	  When enabled, controller will accept Connection Parameter Request
 	  intervals down to a proprietary minimum value.
 
+config BT_CTLR_USER_CPR_ANCHOR_POINT_MOVE
+	bool "Proprietary handling of anchor point movement"
+	depends on BT_CTLR_USER_EXT && BT_CTLR_CONN_PARAM_REQ
+	help
+	  When enabled, defer accept/reject of Connection Parameter Request with only
+	  anchor point movement to proprietary logic.
+
```

Comment:
As far as i can see the accept/reject of anchor point movement is only for the peripheral, so shouldn't that be mentioned and be dependent on it.